src/MockXhr.js

```diff
@@ -168,7 +170,7 @@ MockXhr.prototype.send = function(body) {
       instanceOnSend.call(xhr, xhr);
     }, 0);
   }
-  if (typeof MockXhr.onSend === 'function') {
+  else if (typeof MockXhr.onSend === 'function') {
```

Comment:
This change does not break any tests but it does change the functionality.
I do think that this now handles onSend how what you were explaining in the issue about separate onSend methods. issue #3 .
Also, it is impossible to have separation if the global onSend is called even if an onSend is defined for an instance.